manifests/cli.pp

```diff
@@ -28,16 +28,16 @@
 #                         can be set to specific version number, 'latest', 'present' etc.
 #
 class foreman::cli (
-  Optional[Stdlib::HTTPUrl] $foreman_url = $foreman::cli::params::foreman_url,
-  String $version = $foreman::cli::params::version,
-  Boolean $manage_root_config = $foreman::cli::params::manage_root_config,
-  Optional[String] $username = $foreman::cli::params::username,
-  Optional[String] $password = $foreman::cli::params::password,
-  Boolean $use_sessions = $foreman::cli::params::use_sessions,
-  Boolean $refresh_cache = $foreman::cli::params::refresh_cache,
-  Integer[-1] $request_timeout = $foreman::cli::params::request_timeout,
-  Optional[Stdlib::Absolutepath] $ssl_ca_file = $foreman::cli::params::ssl_ca_file,
-  String $hammer_plugin_prefix = $foreman::cli::params::hammer_plugin_prefix,
+  Optional[Stdlib::HTTPUrl]                    $foreman_url          = $foreman::cli::params::foreman_url,
```

Comment:
I'd prefer not to use alignment. It really creates more problems than it solves.

Reply:
I un-aligned the Parameters again.